jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
+  ]
```

Comment:
I'm inclined to let this bug crop up again if it's a problem. I know it's unfortunate to potentially reintroduce a bug, but it still seems better than applying a fix that might not even be fixing anything. We'd have to figure out what a dnsmasq equivalent would be on Windows, which would require understanding what makes routing DNS traffic through dnsmasq different from just binding Consul to port 53 and using it directly.